### DISCUSSION TITLE
GitHub Token/pipeline

### DISCUSSION THREAD

**Discussion by EdwardCooke** [aaubry/YamlDotNet#749] [General]
@aaubry  I think the github token for the release pipeline is expired. I got the release working, but now it's getting a 403 error from github. Can you update that token? Or is it supposed to be using some kind of token that is auto generated by the pipeline.

**Comment by aaubry** [comment]
I probably need to generate a new token. I'll take care of it later today.

**Reply by aaubry** [reply]
I won't be able to check it today after all, but tomorrow I definitely will.

**Reply by aaubry** [reply]
I've found the issue. It seems that Bullseye was updated to a broken version that doesn't detect correctly the host it is running on. I've already commited the fix on master and created a draft release. Up to you to either publish it or delete it.

**Comment by EdwardCooke** [comment]
Sounds good, thanks. I've been wondering if we consider replacing the build project and just github actions. But that's for a later time.

**Comment by aaubry** [comment]
I have a strong opinion against that. I don't want this project to be tied to GitHub or any other platform. Anyone should be able to clone this project and build it themselves. That includes publishing the NuGet package, generating release notes, etc.
